annotations/source/en/adt/union/union.md

```diff
@@ -275,6 +275,70 @@ all cases when using it:
       })
     });
 
+### Reacting to Many Errors with One Result
+### Reacting To Anything
+### Dealing All Error Scenarios
+### Dealing All Error Scenarios in One Fell Swoop
+### Logging Any Errors
+
+Handling tagged unions like the above example shows we can now handle errors
+in general, yet still retain the ability to react to each individual
+type of error.
+
+In some scenarios, we wish to react to any response, not caring which
+one it is. Examples include finding required HTTP headers for a request.
+Another is when calling noops for log and resource deallocation. For
+those situations, we can use the `any` type.
```

Comment:
Instead of:

```
we can use the `any` type.
```

It would be more accurate/less confusing to use:

```
we can use the special value `any`
```